data/net/minecraft/world/level/levelgen/synth/PerlinSimplexNoise.mapping

```diff
@@ -1,10 +1,15 @@
 CLASS net/minecraft/world/level/levelgen/synth/PerlinSimplexNoise
+	COMMENT This class generates multiple octaves of simplex noise. Each individual octave is an instance of {@link net.minecraft.world.level.levelgen.synth.SimplexNoise}.
+	COMMENT Mojang uses the term 'Perlin' to describe octaves or fBm (Fractal Brownian Motion) noise and this class does not actually generate perlin noise.
```

Comment:
Inconsistent capitalisation of Perlin between Javadocs. I think the capitalised P makes more sense anyway.
```suggestion
	COMMENT Mojang uses the term 'Perlin' to describe octaves or fBm (Fractal Brownian Motion) noise and this class does not actually generate Perlin noise.
```

Reply:
Perlin noise > perlin noise yes as the former is a proper noun (the last name of Ken Perlin) ((Not capitalized because it's a reference to the class))